### PR TITLE
[zh-cn] sync scheduling-eviction/node-pressure-eviction.md

### PR DESCRIPTION
**Pull request by my-git9**
sync content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 1]
```diff
@@ -29,49 +29,87 @@ Node-pressure eviction is not the same as
...
+
+<!--
+The kubelet takes the _priority_ of the static pod into account when creating
+a replacement. If the static pod manifest specifies a low priority, and there
+are higher-priority Pods defined within the cluster's control plane, and the
+node is under resource pressure, the kubelet may not be able to make room for
+that static pod. The kubelet continues to attempt to run all static pods even
+when there is resource pressure on a node.
+-->
+创建替换时，kubelet 会考虑静态 Pod 的优先级。如果静态 Pod 清单指定了低优先级，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:97]:
```suggestion
创建替代 Pod 时，kubelet 会考虑静态 Pod 的优先级。如果静态 Pod 清单指定了低优先级，
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 2]
```diff
@@ -29,49 +29,87 @@ Node-pressure eviction is not the same as
...
+并且集群的控制平面内定义了较高优先级的 Pod，并且节点面临资源压力，则 kubelet
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:98]:
```suggestion
并且集群的控制平面内定义了优先级更高的 Pod，并且节点面临资源压力，则 kubelet
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 3]
```diff
@@ -29,49 +29,87 @@ Node-pressure eviction is not the same as
...
 <!--
+## Eviction signals and thresholds
+
 The kubelet uses various parameters to make eviction decisions, like the following:
 
 - Eviction signals
 - Eviction thresholds
 - Monitoring intervals
 -->
+## 驱逐信号和阈值
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:111]:
```suggestion
## 驱逐信号和阈值   {#eviction-signals-and-thresholds}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 4]
```diff
@@ -132,27 +170,28 @@ memory is reclaimable under pressure.
...
-kubelet 支持以下文件系统分区：
+kubelet 可识别以下两个特定的文件系统标识符：
 
-1. `nodefs`：节点的主要文件系统，用于本地磁盘卷、emptyDir、日志存储等。
+1. `nodefs`：节点的主要文件系统，用于本地磁盘卷、不受内存支持的 emptyDir 卷、日志存储等。
    例如，`nodefs` 包含 `/var/lib/kubelet/`。
 1. `imagefs`：可选文件系统，供容器运行时存储容器镜像和容器可写层。
 
-kubelet 会自动发现这些文件系统并忽略其他文件系统。kubelet 不支持其他配置。
+kubelet 会自动发现这些文件系统并忽略其他节点本地文件系统。kubelet 不支持其他配置。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:194]:
```suggestion
kubelet 会自动发现这些文件系统并忽略节点本地的其它文件系统。kubelet 不支持其他配置。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 5]
```diff
@@ -193,6 +233,7 @@ Eviction thresholds have the form `[eviction-signal][operator][quantity]`, where
 ### 驱逐条件 {#eviction-thresholds}
 
 你可以为 kubelet 指定自定义驱逐条件，以便在作出驱逐决定时使用。
+你可以设置[软](#soft-eviction-thresholds)和[硬](#hard-eviction-thresholds)驱逐阈值。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:236]:
```suggestion
你可以设置[软性的](#soft-eviction-thresholds)和[硬性的](#hard-eviction-thresholds)驱逐阈值。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 6]
```diff
@@ -305,25 +346,26 @@ should provide all the thresholds respectively.
...
 <!--
-### Eviction monitoring interval
+## Eviction monitoring interval
 
-The kubelet evaluates eviction thresholds based on its configured `housekeeping-interval`
+The kubelet evaluates eviction thresholds based on its configured `housekeeping-interval`,
 which defaults to `10s`.
 -->
-### 驱逐监测间隔
+## 驱逐监测间隔
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:354]:
```suggestion
## 驱逐监测间隔   {#eviction-monitoring-interval}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 7]
```diff
@@ -305,25 +346,26 @@ should provide all the thresholds respectively.
...
 
-The kubelet reports node conditions to reflect that the node is under pressure
-because hard or soft eviction threshold is met, independent of configured grace
-periods.
+The kubelet reports [node conditions](/docs/concepts/architecture/nodes/#condition)
+to reflect that the node is under pressure because hard or soft eviction
+threshold is met, independent of configured grace periods.
 -->
-### 节点条件 {#node-conditions}
+## 节点条件 {#node-conditions}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:365]:
```suggestion
## 节点状况 {#node-conditions}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 8]
```diff
@@ -305,25 +346,26 @@ should provide all the thresholds respectively.
...
 
-kubelet 报告节点状况以反映节点处于压力之下，因为满足硬或软驱逐条件，与配置的宽限期无关。
+kubelet 报告[节点状况](/zh-cn/docs/concepts/architecture/nodes/#condition)以反映节点处于压力之下，
+因为满足硬或软驱逐条件，与配置的宽限期无关。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:368]:
```suggestion
原因是满足硬性的或软性的驱逐条件，与配置的宽限期无关。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 9]
```diff
@@ -345,10 +390,12 @@ kubelet 根据下表将驱逐信号映射为节点状况：
 | `DiskPressure`   | `nodefs.available`、`nodefs.inodesFree`、`imagefs.available` 或 `imagefs.inodesFree` | 节点的根文件系统或镜像文件系统上的可用磁盘空间和 inode 已满足驱逐条件 |
 | `PIDPressure`    | `pid.available` | (Linux) 节点上的可用进程标识符已低于驱逐条件 |
 
+控制平面还将这些节点条件[映射]((/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition))到污点。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:393]:
```suggestion
控制平面还将这些节点状况[映射]((/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition))为其污点。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 10]
```diff
@@ -358,7 +405,7 @@ To protect against oscillation, you can use the `eviction-pressure-transition-pe
 flag, which controls how long the kubelet must wait before transitioning a node
 condition to a different state. The transition period has a default value of `5m`.
 -->
-#### 节点条件振荡
+### 节点条件振荡
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:408]:
```suggestion
### 节点状况波动   {#node-condition-oscillation}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 11]
```diff
@@ -487,15 +534,25 @@ will choose to evict pods of lowest Priority first.
...
+the Pods' relative priority to determine the eviction order, because inodes and PIDs have no
 requests.
 
 The kubelet sorts pods differently based on whether the node has a dedicated
 `imagefs` filesystem:
 -->
-当 kubelet 因 inode 或 PID 不足而驱逐 Pod 时，
-它使用优先级来确定驱逐顺序，因为 inode 和 PID 没有请求。
+当 kubelet 因 inode 或 进程 ID 不足而驱逐 Pod 时，
+它使用 Pod 的相对优先级来确定驱逐顺序，因为 inode 和 PID 没有请求。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:555]:
```suggestion
它使用 Pod 的相对优先级来确定驱逐顺序，因为 inode 和 PID 没有对应的请求字段。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 12]
```diff
@@ -569,53 +626,55 @@ evictionMinimumReclaim:
...
 
-If the node experiences an out of memory (OOM) event prior to the kubelet
+If the node experiences an _out of memory_ (OOM) event prior to the kubelet
 being able to reclaim memory, the node depends on the [oom_killer](https://lwn.net/Articles/391222/)
 to respond.
 
 The kubelet sets an `oom_score_adj` value for each container based on the QoS for the pod.
 -->
-### 节点内存不足行为
+## 节点内存不足行为
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:657]:
```suggestion
## 节点内存不足行为   {#node-out-of-memory-behavior}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 13]
```diff
@@ -638,19 +697,19 @@ based on its `RestartPolicy`.
...
+## Good practices {#node-pressure-eviction-good-practices}
 
-The following sections describe best practices for eviction configuration.
+The following sections describe good practices for eviction configuration.
 -->
-### 最佳实践 {#node-pressure-eviction-good-practices}
+### 良好实践 {#node-pressure-eviction-good-practices}
 
-以下部分描述了驱逐配置的最佳实践。
+以下部分描述了驱逐配置的良好实践。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:709]:
```suggestion
以下各小节阐述驱逐配置的好的做法。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 14]
```diff
@@ -679,44 +738,45 @@ For this to work, the kubelet is launched as follows:
...
+### DaemonSets 和节点压力驱逐  {#daemonset}
 
 Pod 优先级是做出驱逐决定的主要因素。
-如果你不希望 kubelet 驱逐属于 `DaemonSet` 的 Pod，
-请在 Pod 规约中为这些 Pod 提供足够高的 `priorityClass`。
-你还可以使用优先级较低的 `priorityClass` 或默认配置，
+如果你不希望 kubelet 驱逐属于 DaemonSet 的 Pod，
+请在 Pod 规约中通过指定合适的 `priorityClassName` 为这些 Pod
+提供足够高的 `priorityClass`。
+您还可以使用较低优先级或默认优先级的 `priorityClass`，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:775]:
```suggestion
你还可以使用较低优先级或默认优先级，以便
```

File: content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 15]
```diff
@@ -679,44 +738,45 @@ For this to work, the kubelet is launched as follows:
...
 仅在有足够资源时才运行 `DaemonSet` Pod。
 
 <!--
-### Known issues
+## Known issues
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/node-pressure-eviction.md:779]: 中文要提级，并添加 anchor。

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**